src/javascript/enhancements/anilyr.ts

```diff
@@ -1,9 +1,13 @@
-import { getGlobalConfiguration, SETTINGS_playerAutoplayAfterScreenshot } from '../configuration/configuration';
+import { getGlobalConfiguration,
+    SETTINGS_playerAutoplayAfterScreenshot,
+    SETTINGS_playerAutopauseAfterFocusLost,
+    SETTINGS_playerAutoplayAfterFocusGain } from '../configuration/configuration';
 import * as core from '../utils/aniwatchCore';
 import * as helper from '../utils/helpers';
 
 const SCREENSHOT_TOOLTIP_ID = 'anilyr-screenshots-tooltip';
 const PLAYER_ID = 'player';
+let onVisible: boolean;
```

Comment:
Maybe would rename it to indicate it's a Boolean. `onSomething` mostly is used for events or listeners.

```suggestion
let restartPlayerOnVisible: boolean;
```

Reply:
I agree, instead of 'restart' I used 'resume' since it fits the overall functionality better.